src/debug-layer/debug-command-encoder.cpp

```
@@ -1849,6 +1849,22 @@ void DebugCommandEncoder::globalBarrier()
+void DebugCommandEncoder::aliasResources(IResource* before, IResource* after)
+{
+    SLANG_RHI_DEBUG_API(ICommandEncoder, aliasResources);
+
+    requireOpen();
+    requireNoPass();
+
+    if (!after)
+    {
+        RHI_VALIDATION_ERROR("'after' must not be null.");
+        return;
+    }
```

Cppcheck (2.21.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)